ds_autotuning_prototype/trial_classes/cifar10_moe_fake_data/README.md

```diff
@@ -0,0 +1,30 @@
+# DeepSpeed CIFAR Example
+This example is adapted from the 
+[CIFAR example in the DeepSpeedExamples](https://github.com/microsoft/DeepSpeedExamples/tree/master/cifar) 
+repository. It is intended to demonstrate a simple usecase of DeepSpeed with Determined.
+
+## Files
+* **model_def.py**: The core code for the model. This includes building and compiling the model.
+
+### Configuration Files
+* **ds_config.json**: The DeepSpeed config file.
+* **moe.yaml**: Determined config to train the model with Mixture of Experts enabled.
+* **zero_stages.yaml**: Same as `moe.yaml`, but trains the model with ZeRO stage 2 optimizer.
+
+## Data
+The CIFAR-10 dataset is downloaded from https://www.cs.toronto.edu/~kriz/cifar.html.
+
+## To Run
+If you have not yet installed Determined, installation instructions can be found
+under `docs/install-admin.html` or at https://docs.determined.ai/latest/index.html
+
+Run the following command: 
+```
+det experiment create moe.yaml .
```

Comment:
Just a nit, but we could also add something about how to call this with the `python3 -m determined.pytorch.deepspeed.dsat.autotune moe.yaml .` style

Reply:
yeah, for sure will do that in all of the examples we edit.  This example is basically scratch work, though.